test/nuxt/composables.test.ts

```diff
@@ -691,6 +692,56 @@ describe('useCookie', () => {
     expect(computedVal.value).toBe(0)
   })
 
+  it('preset test cookie', () => {
```

Comment:
I don't understand why, in the same test case, some cookies are preset but in subsequent test code, I can't get cookies even if I try to add delayed fetch." That's why I preset cookies in separate test cases in advance and then add test cases later (because that's the way to get to the preset cookies). I know this approach isn't very reliable, but after reviewing the project code, I can't think of a better solution. Is there a better solution?